sql/mysqld_thd_manager.cc

```diff
@@ -27,6 +27,7 @@
 #include "wsrep_sst.h"
 #endif /* WITH_WSREP */
 
+#include "mysql/components/services/log_builtins.h"
```

Comment:
If we are referencing something from this and then it would be wsrep specific. 
(Can you move this to WITH_WSREP ?)
and same for all other lines added

Reply:
Moved all the log_builtins.h inside of WITH_WSREP (well those in non-wsrep code files).